src/Build/Graph/ProjectGraph.cs

```diff
@@ -597,6 +606,19 @@ internal string ToDot(Func<ProjectGraphNode, string> nodeIdProvider)
             }
 
             return targetLists;
+
+            void ThrowOnEmptyTargetNames(ICollection<string> targetNames)
+            {
+                if (targetNames == null || targetNames.Count == 0)
+                {
+                    return;
+                }
+
+                if (targetNames.Any(targetName => string.IsNullOrWhiteSpace(targetName)))
+                {
+                    throw new ArgumentException(ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword("OM_TargetNameNullOrEmpty", nameof(GetTargetLists)));
```

Comment:
Is this enough context? I was thinking more along the lines of "what project caused this error?"

Reply:
Ah, that's a bit harder to answer. There is no actual project here. These targets are the graph wide targets the whole graph needs to be built with. The closest thing for identification would be the set of graph roots